tox.ini

```diff
@@ -42,7 +42,7 @@ deps=
 
 commands =
     unit: pytest --maxfail=10 -v --cov=celery --cov-report=xml --cov-report term {posargs}
-    integration: pytest -xsv t/integration {posargs}
+    integration: pytest -xsv t/integration/{posargs}
```

Comment:
As we previously discussed, this is a bad idea since we'd like to pass additional positional arguments to pytest in CI which is more important than running only part of the integration suite using positional arguments. You may pass `-k <test name>` to run a specific test or file. Please revert this.